GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2126,6 +2136,35 @@ internal void ShowFirstParent()
             ForceRefreshRevisions();
         }
 
+        internal void ToggleBetweenArtificialAndHeadCommits()
+        {
+            GoToRef(GetIdToSelect()?.ToString(), showNoRevisionMsg: false);
+            ToggledBetweenArtificialAndHeadCommits?.Invoke(this, EventArgs.Empty);
+            return;
+
+            ObjectId GetIdToSelect()
+            {
+                // try the up to 3 possibilities
```

Comment:
Suggestion to clarify toggle sequence
// Try WorkTree->Index->HEAD

Reply:
will do